common/logger/init.go

```diff
@@ -13,9 +13,9 @@ const (
 	DefaultTimeFormat           = "15:04:05.000"
 	DefaultCallerSkipFrameCount = 3 // set to 3 because logger wrapped in logger.go
 
-	NoColor   = true
-	UseCaller = false // for developer, if you want to expose line of code of caller
-	flagDebug = "log-debug"
```

Comment:
can we make it such that the CLI log level is configurable by passing a parameter like `world cardinal start --cli.log-level=debug`?

Reply:
Yes we can, but i don't think it will be useful. how if we just change the flag name with `--verbose`?

Reply:
sure [](edited-with-Axolo)